ddp-workspace/projects/ddp-singular/playwright-e2e/tests/nav.ts

```diff
@@ -0,0 +1,89 @@
+import { Locator, Page, Response } from '@playwright/test';
+// import dotenv from 'dotenv';
+// dotenv.config();
+
+export const NavSelectors = {
```

Comment:
_**#minor**_

Perhaps, it makes sense to keep the selectors in a separate file because it s going to be bigger in the future.

Reply:
That's a good idea for a large UI. Hopefully we do not create too many selectors in a single file. This file is okay in size.